feeds/it.json

```
@@ -12,7 +12,8 @@
             "mdb-id": "mdb-2668",
             "fix": true,
             "license": {
-                "url":"https://www.dati.gov.it/content/italian-open-data-license-v20"
+                "url":"https://www.dati.gov.it/content/italian-open-data-license-v20",
+                "spdx-identifier": "IODL-2.0"
```

```suggestion
                "spdx-identifier": "LicenseRef-IODL-2.0"
```
There is no standardised identifier for this license, this should work I think